keyboards/kprepublic/bm65hsrgb_iso/keymaps/p4yne/keymap.c

```diff
@@ -0,0 +1,477 @@
+  [_LVL7_] = LAYOUT_65_iso_blocker(
+      KC_ESC,     DE_1,    DE_2,    DE_3,    DE_4,    DE_5,    DE_6,    DE_7,    DE_8,    DE_9,    DE_0,      DE_SS,   DE_ACUT, KC_BSPC, KC_DEL,
+      KC_TAB,     DE_Q,    DE_W,    DE_E,    DE_R,    DE_T,    DE_Z,    DE_U,    DE_I,    DE_O,    DE_P,      DE_UDIA, DE_PLUS,          TD(TD_HOME_END),
+      KC_CAPS,    DE_A,    DE_S,    DE_D,    DE_F,    DE_G,    DE_H,    DE_J,    DE_K,    DE_L,    DE_ODIA,   DE_ADIA, DE_HASH, KC_ENT,  KC_PGUP,
+      KC_LSPO, DE_LABK,    DE_Y,    DE_X,    DE_C,    DE_V,    DE_B,    DE_N,    DE_M,    DE_COMM, DE_DOT,    DE_MINS, KC_RSPC, KC_UP,   KC_PGDN,
+      KC_LCTL, KC_LGUI, KC_LAPO,                    KC_SPC,                               KC_RAPC, TO(_LVL0_),         KC_LEFT, KC_DOWN, KC_RGHT
+      ),
+#else
+  /* 0: ISO qwertz, SPACECADET but no TAPDANCE */
+  [_LVL7_] = LAYOUT_65_iso_blocker(
+      KC_ESC,     KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,      KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,
+      KC_TAB,     KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,      KC_LBRC, KC_RBRC,          KC_HOME,
+      KC_CAPS,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN,   KC_QUOT, KC_NUHS, KC_ENT,  KC_PGUP,
+      KC_LSPO, KC_NUBS,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,    KC_SLSH, KC_RSPC, KC_UP,   KC_PGDN,
+      KC_LCTL, KC_LGUI, KC_LAPO,                    KC_SPC,                               KC_RAPC, TO(_LVL0_),         KC_LEFT, KC_DOWN, KC_RGHT
+      ),
+#endif
+
+
+  /*template
+    [_UL] = LAYOUT_65_iso_blocker(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, TO(0),   _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______,                   _______,                            _______, _______,          _______, _______, _______
+    )
+    */
+};
+
+#ifdef _______
+#undef _______
+#define _______ {0, 0, 0}
+
+
+// layer color and type
+const uint8_t PROGMEM ledmap[][4] = {
+  // LEDs off on Layer 0
+  //[_LVL0_] = _______,
+  [_LVL0_] = {C_HSV_WINERED,   LYR_SOLID},
+  [_LVL1_] = {HSV_RED,         LYR_TRANS},
+  [_LVL2_] = {HSV_MAGENTA,     LYR_BLACK},
+  [_LVL3_] = {HSV_PURPLE,      LYR_BLACK},
+  [_LVL4_] = {C_HSV_DARKGOLD,  LYR_SOLID},
+  [_LVL5_] = {C_HSV_DARKBLUE,  LYR_SOLID},
+  [_LVL6_] = {HSV_BLUE,        LYR_TRANS},
+  [_LVL7_] = {C_HSV_WINERED,   LYR_SOLID}
+};
+
+#undef _______
+#define _______ KC_TRNS
+#endif
```

Comment:
Why go through the process of undefining and redefining `_______` just for a line that's commented out and inactive anyways?